src/pykiso/lib/auxiliaries/udsaux/uds_auxiliary.py

```diff
@@ -271,6 +306,31 @@ def tester_present_sender(self, period: int = 4) -> Iterator[None]:
             stop_event.set()
             sender.join()
 
+    def start_tester_present_sender(self, period: int) -> None:
```

Comment:
doesnt uds have a default period for tester present ?

Reply:
done